webrisk/README.md

```diff
@@ -9,7 +9,7 @@ redirector to redirect users to a warning page for unsafe URLs. The `wrlookup`
 program is a command line service that can also be used to check URLs.
 
 This **README.md** is a quickstart guide on how to build, deploy, and use the
-`safebrowsing` Go package. It can be used out-of-the-box. The GoDoc and API
+`WebRisk` Go package. It can be used out-of-the-box. The GoDoc and API
```

Comment:
Might want the full package path - `cloud.google.com/go/webrisk`. Can also link to the godoc, if you want.

Reply:
Updated